pkg/apis/ceph.rook.io/v1/webhook.go

```diff
@@ -0,0 +1,89 @@
+package v1
+
+import (
+	"github.com/coreos/pkg/capnslog"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/runtime"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"strconv"
+)
+
+var (
+	webhookName = "rook-ceph-webhook"
+	logger      = capnslog.NewPackageLogger("github.com/rook/rook", webhookName)
+)
+
+// compile-time assertions ensures CephCluster implements webhook.Validator so a webhook builder
+// will be registered for the validating webhook.
+var _ webhook.Validator = &CephCluster{}
+
+func (in *CephCluster) ValidateCreate() error {
```

Comment:
Instead of `in`, it's more common to use the first letter of the type.
```suggestion
func (c *CephCluster) ValidateCreate() error {
```